pkgs/by-name/li/libbytesize/package.nix

```diff
@@ -52,6 +55,13 @@ stdenv.mkDerivation (finalAttrs: {
 
   strictDeps = true;
 
+  postInstall = ''
+    substituteInPlace $out/${python3.sitePackages}/bytesize/bytesize.py \
+      --replace 'CDLL("libbytesize.so.1")' "CDLL('$out/lib/libbytesize.so.1')"
+  '';
+
+  pythonImportsCheck = [ "bytesize" ];
```

Comment:
```
       > Running phase: pythonImportsCheckPhase
       > Executing pythonImportsCheckPhase
       > Check whether the following modules can be imported: bytesize
       > Traceback (most recent call last):
       >   File "<string>", line 1, in <module>
       >   File "<string>", line 1, in <lambda>
       >   File "/nix/store/pgb120fb7srbh418v4i2a70aq1w9dawd-python3-3.12.5/lib/python3.12/importlib/__init__.py", line 90, in import_module
       >     return _bootstrap._gcd_import(name[level:], package, level)
       >            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
       >   File "<frozen importlib._bootstrap>", line 1387, in _gcd_import
       >   File "<frozen importlib._bootstrap>", line 1360, in _find_and_load
       >   File "<frozen importlib._bootstrap>", line 1324, in _find_and_load_unlocked
       > ModuleNotFoundError: No module named 'bytesize'
       For full logs, run 'nix log /nix/store/wbjyg7rx7q973wjiqqmilnqf6rkfihh4-libbytesize-2.11.drv
```

Reply:
How do I reproduce this, please? For me, `nix-build -A libbytesize` succeeds with:

```
...
gzipping man pages under /nix/store/8jh4iw2z6j4hbns3f775iif3rrha7qis-libbytesize-2.11-man/share/man/
patching script interpreter paths in /nix/store/8jh4iw2z6j4hbns3f775iif3rrha7qis-libbytesize-2.11-man
Running phase: pythonImportsCheckPhase
@nix { "action": "setPhase", "phase": "pythonImportsCheckPhase" }
Executing pythonImportsCheckPhase
Check whether the following modules can be imported: bytesize
/nix/store/a3v6bdqryzh58zsbfxbypw1ccyy84i8l-libbytesize-2.11
```

The store path seems to be different, too.

Reply:
Figured it out, it looks like we if we want to use `pythonImportsCheck` then we need to also pass in `python3Packages` from `python-packages.nix`.